packages/shared/src/utils/humanize.js

```diff
@@ -73,6 +73,9 @@ const convertBaseValueToUnits = (
   const sliceIndex = initialUnit ? unitArray.indexOf(initialUnit) : 0;
   const units_ = unitArray.slice(sliceIndex);
 
+  let isNegative = false;
+  if (value < 0) isNegative = true;
```

Comment:
`humamizeNegative` pass this optional parameter, default value of this should be `false` so that it won't affect existing graphs where this utility is used...
```suggestion
const shouldHumamizeNegative = humamizeNegative && value < 0;
```

Reply:
we don't need to pass the argument, because this should be the correct if a graph would ever encounter a negative value

Reply:
argument is because we don't want to affect any existing graphs which are using this util... we never had any bugs on those existing graphs, so better to keep their working as it is...

Reply:
I don't think it will affect any graph, the behaviour is the same eg: "100000" would be "12,5KiB" but this change just changes "-100000" -> "-100000B" to "-100000"->"-12.5KiB"

Reply:
let the behaviour same at other places as it is right now, no need to humanize or anything... fix it for the specific case where u really need to based on arg...
`shared` components are not just used by ODF, but other plugins as well... it is a library... don't make any assumptions and add a generic fix (unless really needed) and add optional changes instead...

Reply:
let the behaviour/flow same at other places as it is right now... fix it for the specific case where u really need to based on arg...
`shared` components are not just used by ODF, but other plugins as well... it is a library... don't make any assumptions and add a generic fix (unless really needed) and add optional changes instead...

Reply:
@SanjalKatiyar I agree with you about keeping the current behaviour but the best way to keep peace of mind is to add several unit tests for the utility that ensure the default behaviour as well as the correctness of the new one.